### PR TITLE
Fix pytest version, fix flake8 error, and add pytest-flake8.

### PR DESCRIPTION
**Pull request by somexpert**
ping @ldko, @madhulika95b, this is ready for review.

### CODE REVIEW

**Comment by somexpert** [comment]
Yep, I had deleted the db container and decided to see how this would work without having that container warmed up, and there were a ton of failures, and from what I saw they were all related to the DB connection.

File: README.md [hunk 1]
```diff
@@ -60,7 +62,9 @@ $ docker-compose up -d app
 ## Running the tests
 
 ```sh
-# On the initial run or if the schema changes
+# On the initial run or if the schema changes.
+# The db container must already be running, or the tests will probably
+# fail on the first run since the database needs time to warm up.
```
Comment (ldko) [inline README.md]: Having the explanation in these comments as 2 sentences (one that isn't really a sentence) reads a little awkwardly to me. I would probably move the explanatory text outside of comments, but if you prefer to leave them here, I am ok with that.

File: requirements.txt [hunk 2]
```diff
@@ -6,8 +6,10 @@ MySQL-python==1.2.3rc1
 python-dateutil==1.4.1
```
Comment (ldko) [inline requirements.txt]: I think it makes sense to also bump some other versions:
django==1.8.19
lxml==3.3.0
python-dateutil==2.4

File: README.md (hunk 1 above)
Comment (somexpert) [reply README.md]: done....

File: requirements.txt (hunk 2 above)
Comment (somexpert) [reply requirements.txt]: .....and done

**Pull request merged**